### PR TITLE
MC-40649: Included redirects for old Safe Upgrade Tool links

### PR DESCRIPTION
**Pull request by danidelcar**
## Purpose of this pull request

This pull request (PR) updates the upgrade compatibility tool folder (and pages) with the redirects for old Safe Upgrade Tool links.

## Affected DevDocs pages

This PR modifies the following pages:

- [Introduction page](https://devdocs.magento.com/upgrade-compatibility-tool/introduction.html)
- [Prerequisites page](https://devdocs.magento.com/upgrade-compatibility-tool/prerequisites.html)
- [Install page](https://devdocs.magento.com/upgrade-compatibility-tool/install.html)
- [Run page](https://devdocs.magento.com/upgrade-compatibility-tool/run.html)
- [Developer page](https://devdocs.magento.com/upgrade-compatibility-tool/developer.html)

## Other information

JIRA [MC-40649](https://jira.corp.magento.com/browse/MC-40649)

### CODE REVIEW

**Review by hickskenh — APPROVED** [review]
The redirects look good to me.

**Comment by danidelcar** [comment]
running tests

File: src/upgrade-compatibility-tool/developer.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 group: software-update-guide
 title: Developer information
 ee_only: True
+redirect from:
```
Comment (jeff-matthews) [inline src/upgrade-compatibility-tool/developer.md:5]: you're missing the underscore: `redirect_from`

File: src/upgrade-compatibility-tool/install.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 group: software-update-guide
 title: Install
 ee_only: True
+redirect from:
+  - /safe-upgrade-tool/install.html
```
Comment (jeff-matthews) [inline src/upgrade-compatibility-tool/install.md:6]: you're missing the underscore: `redirect_from`

File: src/upgrade-compatibility-tool/introduction.md [hunk 3]
```diff
@@ -2,6 +2,8 @@
 group: software-update-guide
 title: Introduction
 ee_only: True
+redirect from:
+  - /safe-upgrade-tool/introduction.html
```
Comment (jeff-matthews) [inline src/upgrade-compatibility-tool/introduction.md:6]: you're missing the underscore: `redirect_from`

File: src/upgrade-compatibility-tool/prerequisites.md [hunk 4]
```diff
@@ -2,6 +2,8 @@
 group: software-update-guide
 title: Prerequisites
 ee_only: True
+redirect from:
+  - /safe-upgrade-tool/prerequisites.html
```
Comment (jeff-matthews) [inline src/upgrade-compatibility-tool/prerequisites.md:6]: you're missing the underscore: `redirect_from`

File: src/upgrade-compatibility-tool/run.md [hunk 5]
```diff
@@ -2,6 +2,8 @@
 group: software-update-guide
 title: Run the tool
 ee_only: True
+redirect from:
+  - /safe-upgrade-tool/run.html
```
Comment (jeff-matthews) [inline src/upgrade-compatibility-tool/run.md:6]: you're missing the underscore: `redirect_from`

**Review by jeff-matthews — COMMENTED** [review]
@danidelcar, redirects are not working because the syntax is wrong. See #8616.

Don't forget to test redirects locally or on staging in the future.

**Pull request merged**